lib/sqlalchemy/dialects/mysql/__init__.py

```diff
@@ -7,6 +7,7 @@
 
 from . import base  # noqa
 from . import cymysql  # noqa
+from . import mariadb # noqa
```

Comment:
At least for me, this line borks all attempts to run pytest against any mysql dburi. Even a test against `mysql+pymysql` produces

> INTERNALERROR> sqlalchemy.exc.NoSuchModuleError: Can't load plugin: sqlalchemy.dialects:mysql.pymysql

If I comment out the line then the tests for `mysql+pymysql` run as expected. I tried renaming it to "mariadbconnector" but that didn't help. I suspect that an exception is being raised when this file is being `__import__`ed, but the PyCharm debugger couldn't see it.

Reply:
The exception is

> ImportError: No module named 'mariadb.constants'; 'mariadb' is not a package"

I get the same exception from the console:

``` text
(venv) C:\Users\Gord\git\sqlalchemy>python
Python 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 22:45:29) [MSC v.1916 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import mariadb
>>> mariadb.__version__
'1.0.0'
>>> import mariadb.constants.CLIENT as CLIENT
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'mariadb.constants'; 'mariadb' is not a package
>>>
```

Reply:
Can you please install mariadb-1.0.1 (released yesterday - it contains all changes/fixes for SQLAlchemy)

Reply:
That helped, thanks.